internal/tools/tools.go

```diff
@@ -1,6 +1,6 @@
 package tools
 
 import (
-	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
+	_ "github.com/golangci/golangci-lint/v2/cmd/golangci-lint"
```

Comment:
what is this used for? could this be removed?

golangci-lint supports sorting go imports now, I'm guessing everyone is using golangci-lint directly